test/claim.test.ts

```diff
@@ -0,0 +1,157 @@
+import { DUST_AMOUNT, ONE_ALPH, web3 } from '@alephium/web3'
+import { getSigner } from '@alephium/web3-test'
+import { PrivateKeyWallet } from '@alephium/web3-wallet'
+import { ClaimFee } from '../artifacts/ts'
+import { balanceOf, deployInvariant, expectError } from '../src/utils'
+import { LIQUIDITY_SCALE, PERCENTAGE_SCALE } from '../src/consts'
+import { getPool, initPool, initFeeTier, initPositionWithLiquidity, initTokens, withdrawTokens, initSwap, getPosition } from '../src/testUtils'
+
+web3.setCurrentNodeProvider('http://127.0.0.1:22973')
+let admin: PrivateKeyWallet
+let positionOwner: PrivateKeyWallet
+
+
+describe('invariant tests', () => {
+  beforeAll(async () => {
+    admin = await getSigner(ONE_ALPH * 1000n, 0)
+    positionOwner = await getSigner(ONE_ALPH * 1000n, 0)
+  })
+
+  test('claim', async () => {
+    // init dex and tokens
+    // 1%
+    const protocol_fee = 10n ** (PERCENTAGE_SCALE - 2n)
+    const invariant = await deployInvariant(admin, protocol_fee)
+
+    // 0.6%
+    const fee = 6n * 10n ** (PERCENTAGE_SCALE - 3n)
+    const tickSpacing = 10n
+    await initFeeTier(invariant, admin, fee, tickSpacing)
+
+    let supply = 10n ** 6n + 1000n
+    let amount = 10n ** 6n
+    const [token0, token1] = await initTokens(admin, supply)
```

Comment:
We use the concept of  token namings X and Y when they are sorted by address and 0 and 1 when they are not sorted. So in this case ti should be `tokenX` and `tokenY`